src/Core/Users/Userlogin.php

```diff
@@ -0,0 +1,158 @@
+<?php
+namespace Meetup\Login\Core;
+
+use Meetup\Login\Core\Users\UserInterface;
+
+interface Userlogin implement UserInterface
+{
+
+    /**
+     * Set first name
+     *
+     * @param string $firstName
+     *
+     * @return void
+     */
+    public function setFirstName($firstName){
+        $this->firstname=$firstName;
```

Comment:
As far as i know, this cannot be done in any of the programming languages that support object oriented programming. Can you please enlighten me how does this even remotely work for you?